src/eval-frame/actions/actions.js

```diff
@@ -55,13 +42,6 @@ export function setConsoleLanguage(language) {
   };
 }
 
-export function updateUserVariables() {
```

Comment:
action creator no longer needed; this is handled with a direct response in port-to-editor